### PR TITLE
telegram message

### PR DESCRIPTION
**Pull request by techsmart1022**

### CODE REVIEW

File: config/telegramBotHandler.js [hunk 1]
```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
...
+        //     this.lotteries[chatId].sendMessage(this.startMessage, keyboards);
+        //     // Send the "hello world" message when /new_lottery is triggered
+        //     this.commandStatus[chatId] = 'new_lottery';
+        // });
+        // this.bot.on('callback_query', async (query) => {
+        //     const chatId = query.message.chat.id;
+        //     const data = query.data;
+        //     const lottery = this.lotteries[chatId];
 
-            switch(data){
```
Comment (CodeMaster1022) [inline config/telegramBotHandler.js]: Please fix this part.
Switch status is not correct.

File: config/telegramBotHandler.js [hunk 2]
```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
         });
 
         // Handle the /new_lottery command
-        this.bot.onText(/\/new_lottery/, async (msg) => {
```
Comment (CodeMaster1022) [inline config/telegramBotHandler.js]: When user enter the telegram, userID has to be catched.

File: config/telegramBotHandler.js [hunk 3]
```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
...
-            this.lotteries[chatId].reset(chatId);
-            this.lotteries[chatId].sendMessage(this.startMessage, keyboards);
-            // Send the "hello world" message when /new_lottery is triggered
-            this.commandStatus[chatId] = 'new_lottery';
-        });
-        this.bot.on('callback_query', async (query) => {
-            const chatId = query.message.chat.id;
-            const data = query.data;
-            const lottery = this.lotteries[chatId];
+        // this.bot.onText(/\/new_lottery/, async (msg) => {
```
Comment (CodeMaster1022) [inline config/telegramBotHandler.js]: Okay.
This is good.

File: config/telegramBotHandler.js [hunk 4]
```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
...
-                            this.inputStatus[chatId] = 'CONFIRM';
-                            await this.lotteries[chatId].publishLottery();
-                        }
-                    }
-                    break;
-                case 'USDT':
-                    this.inputStatus[chatId] = 'USDT';
-                    bot.sendMessage(chatId, 'Insert the price of a single ticket in USDT');
-                    break;
-                case 'TON':
```
Comment (CodeMaster1022) [inline config/telegramBotHandler.js]: Ton wallet is not needed.

File: config/telegramBotHandler.js [hunk 5]
```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
...
-                    });
-                    break;
-                case 'RESET':
-                    if (this.lotteries[chatId]) {
-                        this.lotteries[chatId].reset();
-                        this.lotteries[chatId].sendMessage(this.startMessage, keyboards);
-                        this.inputStatus[chatId] = 'RESET';
-                    }
-                    break;
-                case 'CONFIRM':
```
Comment (CodeMaster1022) [inline config/telegramBotHandler.js]: Confirm is not needed.

File: config/telegramBotHandler.js [hunk 6]
```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
...
+        //             this.inputStatus[chatId] = 'ERC';
+        //             bot.sendMessage(chatId, 'Insert the price of a single ticket in ERC');
+        //             break;
+        //         case 'SOL':
+        //             this.inputStatus[chatId] = 'SOL';
+        //             bot.sendMessage(chatId, 'Insert the price of a single ticket in SOL');
+        //             break;
+        //     }
+        // });
         this.bot.on('message', async (msg) => {
```
Comment (CodeMaster1022) [inline config/telegramBotHandler.js]: Add the admin bot.

**Review by CodeMaster1022 — CHANGES_REQUESTED** [review]

File: config/telegramBotHandler.js (hunk 1 above)
Comment (CodeMaster1022) [reply config/telegramBotHandler.js]: Here.

Comment (StackAurora) [reply config/telegramBotHandler.js]: I fixed.

File: config/telegramBotHandler.js (hunk 2 above)
Comment (StackAurora) [reply config/telegramBotHandler.js]: I fixed.

File: config/telegramBotHandler.js (hunk 3 above)
Comment (StackAurora) [reply config/telegramBotHandler.js]: I fixed.

File: config/telegramBotHandler.js (hunk 4 above)
Comment (StackAurora) [reply config/telegramBotHandler.js]: I fixed.

File: config/telegramBotHandler.js (hunk 5 above)
Comment (StackAurora) [reply config/telegramBotHandler.js]: I fixed.

File: config/telegramBotHandler.js (hunk 6 above)
Comment (StackAurora) [reply config/telegramBotHandler.js]: I fixed.

**Pull request merged**